listenbrainz/webserver/views/playlist_api.py

```diff
@@ -55,6 +51,20 @@ def validate_playlist(jspf, require_title=True):
         except KeyError:
             log_raise_400("JSPF playlist must contain a title element with the title of the playlist.")
 
+    try:
+        public = jspf["playlist"]["extension"]["https://musicbrainz.org/doc/jspf#playlist"]["public"]
+        if not isinstance(public, bool):
+            log_raise_400("JSPF playlist public field must contain a boolean.")
+    except KeyError:
+        pass
+
+    try:
+        for collaborator in jspf["playlist"]["extension"]["https://musicbrainz.org/doc/jspf#playlist"]["collabortors"]:
```

Comment:
spelling in this field. should have a test to catch this

Reply:
We have tests. They are spelled correctly. :)